pilot/pkg/networking/plugin/authz/rbac.go

```diff
@@ -557,6 +560,23 @@ func convertRbacRulesToFilterConfig(service *serviceMetadata, option rbacOption)
 	return &http_config.RBAC{Rules: rbac}
 }
 
+// addToRules adds a |rule| to |rules| if |rule| is not nil.
+// |rule| can be a regular rule (e.g. methods) or a not rule (e.g. not_methods).
+func addToRules(rules *policyproto.Permission_AndRules, rule *policyproto.Permission, isNotRule bool) {
```

Comment:
Maybe split this to two functions: `appendRule` and `appendNotRule` to avoid using the third bool parameter.

Reply:
Sounds good. Will make the change.